README.md

```diff
@@ -0,0 +1,38 @@
+# Zuul Config Job
+Provide instruction for using the `Zuul` job in this repo.
+
+###  trigger-build
+This job allows user to invoke the `OpenShift` image generic webhook to rebuild the image. Here are the instructions for properly use this `Zuul` job in your project.
+
+**First Step**
+Here are the three parameters that is required in the `.zuul.yaml` file for determining which webhook your would like to invoke.
+- cluster (optional default is set to `paas.psi.redhat.com`)
+- namespace (requird)
+- buildConfigName (required)
+
+For example, here is an example about how to use this `Zuul` job in `.zuul.yaml` file.
+```yaml
+post:
+  jobs:
+    - "trigger-build":
+      vars:
+        cluster: "paas.psi.redhat.com"
+        namespace: "thoth-test-core"
+        buildConfigName: "user-api"
+```
+
+**Second Step**
+Make sure in the `buildConfig-template.yaml` file, the `generic` trigger is added.
```

Comment:
rather than referring to the `buildConfig-template.yaml` file you could mention the BuildConfig itself, this way it does not depend on our naming scheme

Reply:
Got it!